scripts/irods/test/test_igroupadmin.py

```diff
@@ -72,6 +71,61 @@ def tearDownClass(self):
             self.admin.__exit__()
             admin_session.assert_icommand(['iadmin', 'rmuser', self.admin.username])
 
+    def test_create_user_with_password_in_igroupadmin__issue_6887(self):
+        USERS = { 'alice': 42,
+                  'bob':   40,
+                  'eve':   3   }
```

Comment:
Is there something special about these password lengths?

Reply:
They range from the supposed minimum to maximum, but ... it may not be necessary here to test that range .  If for example a minimum length is something we want to enforce (though I think not) it should be another issue, yes?

Reply:
So maybe just test that the long one works, and be done with it, maybe. @alanking - Let me know what you think.

Reply:
I'm going to recommend that the test focus on just the one issue and you create a new/separate test that handles invalid password lengths.

That way the test names can be clear about what they are covering.

Reply:
ok.  Think I'd need guidance on how to write that issue if I were to do it, though... 
  - It's not always clear for me, when reading the code, whether a length field is a reference to a hashed password length (which in PAM's case I know can be longer than 1024 chars) or the raw one entered by the user on (for example) the passwd, ipasswd or iinit command line.  I'd require deeper knowledge to start writing about the problem. (If there is one ;) ).
  - Do we need a placeholder issue, or are there specific concerns where we think iRODS should be determining / limiting what is a "good" password length.    https://github.com/irods/irods/pull/6889#discussion_r1100205814

Reply:
Let's create a placeholder issue. It can say something like "Add tests for invalid passwords". It doesn't need to go into great detail. It is simply an indicator that we need to investigate the space and improve coverage.

We can revisit it later and this PR gets to move forward.

Reply:
With #6911, I think we can consider this addressed. Thanks!